guidelines/frontend-coding-standard/avoid-double-negations.md

```diff
@@ -0,0 +1,23 @@
+# Avoid any context in double negations
+
+Avoid using if not statement for conditional logic.
+
+```js
+const disabledSomething = true;
+const denyFeeatureA = false;
+const avoidDoingX = true;
+
+// 🤔 avoid this
+if ((!disabledSomething && !denyFeeatureA) || !avoidDoingX) {
+}
+
+const enabledSomething = false;
+const approveFeeatureA = true;
+const shouldDoX = false;
+
+// 👍🏻 Suggest this
+if ((enabledSomething && approveFeeatureA) || shouldDoX) {
+}
+```
```

Comment:
Our discussion:
* not avoid double negations, but avoid naming thing in its negative meaning.